AdSecGHTests/Helpers/ToolStripMenuHelperTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using AdSecGH.Helpers;
+
+using Xunit;
+
+namespace AdSecGHTests.Helpers {
+
+  public class ToolStripMenuHelperTests {
+    [Fact]
+    public void CreateInvisibleMenuItem_ShouldReturnToolStripMenuItem_WithNotAvailableText() {
+      var result = ToolStripMenuHelper.CreateInvisibleMenuItem();
+      Assert.NotNull(result);
+      Assert.Equal("Not available", result.Text);
+    }
+
+    [Fact]
+    public void CreateInvisibleMenuItem_ShouldReturnToolStripMenuItem_WithVisibleEqualFalse() {
+      var result = ToolStripMenuHelper.CreateInvisibleMenuItem();
+      Assert.False(result.Visible);
+    }
+
+    [Fact]
+    public void CreateInvisibleMenuItem_ShouldReturnNewInstanceEachTime() {
+      var result1 = ToolStripMenuHelper.CreateInvisibleMenuItem();
+      var result2 = ToolStripMenuHelper.CreateInvisibleMenuItem();
+      Assert.NotSame(result1, result2);
+    }
```

Comment:
do we need this? I think this is simply checking that two instances are different, it is always true that, unless we have written code to compare them.